src/transaction/log_recovery_analysis.cpp

```diff
@@ -1045,8 +1103,21 @@ log_rv_analysis_complete (THREAD_ENTRY *thread_p, int tran_id, LOG_LSA *log_lsa,
       // Transaction is completed.
       if (tran_index != NULL_TRAN_INDEX)
 	{
+	  // newer quick fix on top of older quick fix: mark the mvccid as completed
```

Comment:
This piece of code (to complete the transaction) is identical to the one in line 1046.

Reply:
extracted function